tests/integration/test_integration.py

```diff
@@ -32,6 +34,15 @@
 
 INTEGRATION_ROOT = Path(__file__).resolve().parent
 
+
+def _marker(model: str, config: DictConfig):
+    """Add markers at collection time based on the model test config."""
+    marks = []
+    if "cosmo" in model or config.get("cosmo"):
+        marks.append(pytest.mark.cosmo)
+    return marks
```

Comment:
Not a big fan of this one. I would prefer if we had an explicit `markers` entry in the configs.

Reply:
Good idea, I changed it in c0a5b772c2b0e2568c7e9a9247046bb583d0b5a1